apps/condo/domains/notification/tasks/deliverMessage.js

```diff
@@ -51,22 +60,22 @@ async function _sendMessageByAdapter (transport, adapter, messageContext) {
         return [true, { fakeAdapter: true, transport, messageContext }]
     }
 
-    return await adapter.send(messageContext)
+    return await adapter.send(messageContext, isVoIP)
```

Comment:
Are there only two message types: "VoIP" and "no-VoIP"?
Should we add another boolean variable if we add another message type?
I mean maybe it's better to add a string variable instead of a boolean one.

Reply:
By now such push providers as Firebase and Huawei support only one push type - ordinary, and Apple supports both ordinary and VoIP push types, and there are no other providers that we plan to use. So changes like suggested look like over engineering imho.